stingray/tests/test_powerspectrum.py

```diff
@@ -236,14 +255,6 @@ def test_rebin_log(self, df):
                                     norm="Leahy", dt=self.dt)
         bin_aps = aps.rebin_log(df)
 
-    def test_rebin_with_invalid_type_attribute(self):
-        new_df = 2
-        aps = AveragedPowerspectrum(self.lc, segment_size=1,
-                                    norm='leahy', dt=self.dt)
-        aps.type = 'invalid_type'
-        with pytest.raises(AttributeError):
-            assert aps.rebin(df=new_df)
```

Comment:
Why does this test get deleted?

Reply:
Because the `type` attribute is now hardcoded into `Powerspectrum`